src/components/issues/TaskRequestForm.tsx

```diff
@@ -0,0 +1,144 @@
+import { FC, MouseEvent, useReducer, useState } from 'react';
+
+import styles from '@/components/issues/Card.module.scss';
+
```

Comment:
Nit: Can you remove the extra line

Reply:
Removed it in PR : #958 

[Click here](https://github.com/Real-Dev-Squad/website-status/pull/958/commits/15d8b5ddeccbb25d6cb579d0f2d368f01844cc66) to view the changes.